### DISCUSSION TITLE
Version 1.1.0

### DISCUSSION THREAD

**Discussion by ahadas** [mucommander/mucommander#748] [General]
As [we completed everything that was targeted for the next version](https://github.com/mucommander/mucommander/milestone/12), I plan to release v1.1.0 in ~two weeks.

Any help with testing the [1.1.0 nightly version](https://github.com/mucommander/mucommander/releases/tag/nightly) would be appreciated! (to see we don't have regressions like #741, thanks @Andreas0602!)

@pskowronek, how about polishing #733? is there something else you work on for v1.1.0?

**Comment by pskowronek** [comment]
replied in #733, for 1.1.0 I don't have anything more (I will update #744 soon).

**Reply by ahadas** [reply]
thanks for all the hard work @pskowronek ! please note that I won't be available much during the next 2 weeks but I'll follow up with you on #733 before releasing 1.1.0

**Comment by ahadas** [comment]
@danielerez @pskowronek @Andreas0602, can you please check if the signed dmg of [v1.1.0](https://github.com/mucommander/mucommander/releases/tag/1.1.0-1) works fine on your machine?

**Reply by ghost** [reply]
It works fine on my machine.

**Reply by pskowronek** [reply]
here, too :)

**Reply by ahadas** [reply]
thanks for the confirmation!

**Comment by pskowronek** [comment]
@ahadas I guess `readme.txt` and `about` must be updated to mention JediTerm as Library.

**Reply by ahadas** [reply]
yep, definitely - can you please post a PR?

**Reply by pskowronek** [reply]
https://github.com/mucommander/mucommander/pull/765

For future improvement: 1) about should be fed from `package/readme.txt` 2) not sure if those licence files laying around in multiple copies shouldn't be removed and only a single instance be kept in the root.

**Reply by ahadas** [reply]
@pskowronek thanks, commented on the PR
I'm not sure I understand (1) ("about should be fed from package/readme.txt") - did you mean how about generating other text from the readme file?
about (2), yeah - I tried to add symlinks for the license file but that didn't go so well with git on Windows. I think it makes sense to reduce the amount of files that describe the license but we'll need to change the packaging configuration to cope with that

**Reply by pskowronek** [reply]
> @pskowronek thanks, commented on the PR I'm not sure I understand (1) ("about should be fed from package/readme.txt") - did you mean how about generating other text from the readme file? 

For example load `package/readme.txt`, parse it to get list of libraries (or at least keep those libs in a file not in the code).
